app/src/main/res/layout/activity_movie_details.xml

```diff
@@ -0,0 +1,297 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ScrollView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_height="match_parent"
+    android:layout_width="match_parent">
+
+
```

Comment:
@atashandroid Please avoid these white spaces and format this file using a shortcut (CTRL+ALT+L / Command+Option+L)